scripts/ci/configure-xsite.sh

```diff
@@ -8,7 +8,7 @@ KIND_KUBEAPI_PORT=6443
 METALLB_VERSION=v0.9.6
 TESTING_NAMESPACE=${TESTING_NAMESPACE-namespace-for-testing}
 KIND_SUBNET=${KIND_SUBNET-172.172.0.0}
-SERVER_IMAGE=${SERVER_IMAGE:-'quay.io/infinispan/server:13.0'}
+SERVER_IMAGE=${SERVER_IMAGE:-'quay.io/infinispan/server:14.0'}
```

Comment:
Good job this is only for loading the latest image into `kind` :see_no_evil:

Reply:
Also, I was thinking about upgrading `METALLB` but since everything is working, I just left as is :rofl: